server/src/main/java/com/project/cisco/database/entity/Language.java

```diff
@@ -0,0 +1,23 @@
+package com.project.cisco.database.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.FieldNameConstants;
+
+@Data
+@ToString(callSuper = true)
+@FieldNameConstants
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Please add here also this annotation:
```
@Table(name = "language",
        uniqueConstraints = {@UniqueConstraint(columnNames = "language")})
```

because now we can create many entity with the same language name

Reply:
Done